### ISSUE TITLE
Test 4, X button to close main menu cannot be reached by keyboard

### ISSUE THREAD

**Issue by david-iterators** [CityOfBoston/Iterators#104]
Screengrab/videos (insert file names here)
2020 03 17, Test 4, X button to close main menu cannot be reached by keyboard.mp4

Link and Node #
https://d8-stg.boston.gov/

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Windows Laptop > Windows 10 > Edge 80

Trusted Tester Test
4. Keyboard Access and Focus

Brief Description
The user can open an important hamburger menu via the icon in the top left of the page. When the menu is open, a large “X” button appears to the menu’s right; it’s easy to click this and close the window with a mouse, but there is no easy way to focus the X element using Tab or Shift + Tab; users limited to keyboard access cannot close the menu.

Steps to reproduce
1.	00:00, Go to the home page and set focus to the URL
2.	00:03, Begin hitting the Tab key until the Menu icon is in focus, and continue pressing the Tab key
3.	00:10, Iterate through the menu options with Tab until well past X button on the page (0:10)
4.	00:15, Use Shift + Tab to back up past X; alternate between Tab and Shift + Tab in this area
a.	Note:  unable to use the keyboard to close the menu

Desired Result
User can reach and activate X button to close the menu using only Tab and Shift keys.

Actual Result
User not able to focus X element using only the keyboard.

[2020 03 17, Test 4, X button to close main menu cannot be reached by keyboard.mp4.zip](https://github.com/CityOfBoston/Iterators/files/4370703/2020.03.17.Test.4.X.button.to.close.main.menu.cannot.be.reached.by.keyboard.mp4.zip)

**Comment by david-iterators** [comment]
Verified on https://d8-uat.boston.gov
